File: bindings/kotlin-native/src/nativeMain/kotlin/org/maplibre/nativeffi/error/UnsupportedFeatureException.kt
package org.maplibre.nativeffi.error

/** Error for unsupported platforms, backends, entry points, or requested behavior. */
public class UnsupportedFeatureException
internal constructor(nativeStatusCode: Int, diagnostic: String = "") :
  MaplibreException(MaplibreStatus.UNSUPPORTED, nativeStatusCode, diagnostic)

File: bindings/dotnet/src/Maplibre.Native/Error/UnsupportedFeatureException.cs
namespace Maplibre.Native.Error;

/// <summary>Exception for unavailable platforms, backends, entry points, or behaviors.</summary>
public sealed class UnsupportedFeatureException : MaplibreException
{
    public UnsupportedFeatureException(
        MaplibreStatus status,
        int? rawStatus,
        string diagnostic,
        Exception? innerException
    )
        : base(status, rawStatus, diagnostic, innerException) { }
}
